ETL/Provision/aws_infrastructure/main.tf

```diff
@@ -0,0 +1,57 @@
+provider "aws" {
+    region = var.AWS_REGION
+    access_key = var.ACCESS_KEY
+    secret_key = var.SECRET_ACCESS_KEY 
+}
+
+resource "aws_iam_role" "provision_lambda_role" {
+  name = "lambda_execution_role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Principal = {
+          Service = "lambda.amazonaws.com"
+        }
+        Effect = "Allow"
+      }
+    ]
+  })
+}
+
+resource "aws_ecr_repository" "provision-repo" {
+  name                 = "hermes-provision-lambda"
+  image_tag_mutability = "MUTABLE"
+  image_scanning_configuration {
+    scan_on_push = false
+  }
+}
```

Comment:
Not needed as terraform doesn't allow to add images

Reply:
```suggestion

```

Reply:
done